### ISSUE TITLE
When selecting code on Firefox, code becomes unreadable.

### ISSUE THREAD

**Issue by TukuToi** [PrismJS/prism#3019]
When on Firefox this preset https://prismjs.com/download.html#themes=prism-twilight&plugins=line-numbers+autoloader+toolbar+copy-to-clipboard makes it impossible to read the code, when we select code chunks with the Mouse.

Looks just fine on all other browsers.
This is confirmed by 3 different people on 3 different computers all using Firefox in distinct versions and OS.
See images for clarification

Any idea what this is due to?
Do other presets have this same issue?

Note, we already tried to replicate the same issue on the very preset demo site, which results in a unreplicable issue. However, note that the preview on https://prismjs.com/download.html#themes=prism-twilight&plugins=line-numbers+autoloader+toolbar+copy-to-clipboard looks very far from reality since it does not use PHP as the code basis to highlight. Selection color is also totally different than what I see in reality when implementing that preview.

Perhaps there is something else that needs to be done to make selected code look good on all browsers?

Looking good before it is selected:
<img width="998" alt="Hot highlighted  Good!" src="https://user-images.githubusercontent.com/11800236/127648009-7fd1c0be-a5ec-4aa6-b8d1-11de610ee048.png">

Looking bad when it is selected on Firefox. Only on Firefox.
<img width="1057" alt="As soon we hihglight with the mouse" src="https://user-images.githubusercontent.com/11800236/127648034-3cf9c7ec-a13e-4337-b679-6ff60a20a93a.png">

**Comment by RunDevelopment** [comment]
Cannot reproduce in FF 89 on Win10.

![image](https://user-images.githubusercontent.com/20878432/127651466-fed450f0-16fc-4fa8-b4aa-32e36cb788a3.png)

My guess is that you are importing multiple themes. The download links you included say that you use the theme "Twilight" but the theme in your screenshots is clearly "Tomorrow Night".

**Comment by TukuToi** [comment]
Sorry the link was wrong.
Indeed it is this https://prismjs.com/download.html#themes=prism-tomorrow&languages=markup+css+clike+javascript&plugins=line-numbers+autoloader+toolbar+copy-to-clipboard
(Tomorrow night)

I am sure I import just one theme, the one linked above, with the plugins selected (all languages).
See screenshots
<img width="302" alt="Screenshot 2021-07-30 at 20 04 30" src="https://user-images.githubusercontent.com/11800236/127656993-1bde3f81-40bf-4b70-b1fe-361a823d1a6b.png">
<img width="284" alt="Screenshot 2021-07-30 at 20 04 18" src="https://user-images.githubusercontent.com/11800236/127656999-9842f7e7-3f30-465a-9a57-dbdad7a0f6d0.png">

Since several people can see this, it is clearly replicable, but perhaps only in the specific instance or the particular way how its used.
Since you can't replicate, I guess I will have to debug it. Will feedback to share what I find.

**Comment by TukuToi** [comment]
Clearly there is something off with the selection highlighting in that theme on Firefox.
I was able to fix it with this CSS:
```
::-moz-selection { /* Code for Firefox */
  color: red;
  background: yellow;
}
```
(the colors are of course just as an example)

But, while this properly applies only to FF, and did not change Safari look (which does not feature this issue), it ruined Chrome - which also did not have any issue with the theme.

Clearly, Firefox handles selection highlighting much different than other browsers, not only is the color always different but also the "lines" get a separator when selecting them, while in Safari and Chrome, they get selected with a "block" and no lines are visible.

Instead of wasting time with this quirk I just used the Standard Theme, which interestingly does not have any issue!

Thus for me case solved, but something is weird with the specific Theme, even if it is not replicable on the very demo site I was able to replicate the issue on a clean slate just adding the specific theme... Anyway - for me case closed.
